docs/admin_api/user_admin_api.rst

```diff
@@ -65,22 +65,23 @@ It returns a JSON body like the following:
     {
         "users": [
             {
-                "name": "<user_id1>",
+                "user_id": "<user_id1>",
```

Comment:
I'm afraid changing this field still violates the backward compatibility of this API, requiring a version bump.

I agree that this should probably be `user_id`, but it'll still require a v3 users API to accomplish.